team_token.go

```diff
@@ -41,6 +41,7 @@ type TeamToken struct {
 	Description string    `jsonapi:"attr,description"`
 	LastUsedAt  time.Time `jsonapi:"attr,last-used-at,iso8601"`
 	Token       string    `jsonapi:"attr,token"`
+	ExpiredAt   time.Time `jsonapi:"attr,expired-at,iso8601"`
```

Comment:
Same comments here as for the Org tokens: We will also need to add a way to pass an `ExpiredAt` to a [`Create` function](https://github.com/hashicorp/go-tfe/pull/672/files#diff-3ba98d05361a24bc7ebd017c5c2a2ee07efc7d4df09d2bfe2d57e105eb411332R23)